incubator/binding-risingwave/pom.xml

```diff
@@ -196,6 +196,7 @@
         <configuration>
           <excludes>
             <exclude>io/aklivity/zilla/runtime/binding/risingwave/internal/types/**/*.class</exclude>
+            <exclude>net.sf.jsqlparser.parser.*</exclude>
```

Comment:
I think this is supposed to be a path pattern instead of a package pattern, right?

Reply:
Actually both work but I changed it to path pattern for consistency.